sql/moz-fx-data-shared-prod/firefox_accounts/docker_fxa_customs_sanitized/view.sql

```diff
@@ -18,7 +18,7 @@ SELECT
     ) AS labels
   ) AS resource,
   textPayload,
-  STRUCT(jsonPayload.block, jsonPayload.op, jsonPayload.action) AS jsonPayload,
+  STRUCT(jsonPayload.block, jsonPayload.op, jsonPayload.action, jsonPayload.errno) AS jsonPayload,
```

Comment:
[Per Wil Clouser](https://mozilla.slack.com/archives/C4D5ZA91B/p1696012479545839?thread_ts=1694704422.853149&cid=C4D5ZA91B), `jsonPayload.foundin` should also be included:
> It looks like we also use jsonpayload.errno and jsonpayload.foundin.  That looks like the last two fields that aren't rendering in this dashboard.